### DISCUSSION TITLE
WAHA PROBLEMS DELETING MESSAGES SENT TO OWN NUMBER OF ACTIVE SESSION

### DISCUSSION THREAD

**Discussion by synapsevzla** [devlikeapro/waha#687] [General]
Greetings everyone I am trying to delete messages sent with waha core, but currently it is not possible to delete messages sent to my own phone number with reminders to keep my own inbox free, because at the end of the message id has an _out example true_584127584825%40c.Us_aaa_out this returns an empty object as a result.

![image](https://github.com/user-attachments/assets/044fcbb4-1493-4412-b45c-9e90b7280272)

What causes messages not to be deleted

**Comment by Kassiosky** [comment]
Same here using n8n.

**Waha Version:** 2025.5.1
**N8N Version:** 1.91.3
**Docker version** 28.1.1, build 4eba377

Log:
`[13:35:41.448] INFO (1): request completed {"reqId":1266,"req":{"id":1266,"method":"DELETE","url":"/api/default/chats/1****34197416*****@g.us/messages/true_1203*****41677565@g.us_3EB0B8****F0DE******D5","query":{},"params":{"0":"api/default/chats/1****34197416*****@g.us/messages/true_1203*****41677565@g.us_3EB0B8****F0DE******D5"}},"res":{"statusCode":200},"responseTime":12}`

Output:
```
[
""
]
```

**Reply by devlikepro** [reply]
Hi! When you remove a message **from a group** - it must be in format 
`{fromMe}_{groupId}_{messageId}_{participant}`

There's `participant` part in your request

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
